lib/config.js

```diff
@@ -320,21 +316,16 @@ class Config {
             normalize(conf, agent.logger)
 
             for (const [key, value] of Object.entries(conf)) {
-              const validator = VALIDATORS[key]
-              if (validator ? validator(value) : true) {
-                const oldValue = agent._conf[key]
-                agent._conf[key] = value
-                if (key === 'logLevel' && value !== oldValue && !logging.isLoggerCustom(agent.logger)) {
-                  // If a new log level value was provided, and we aren't
-                  // already using a custom-provided logger (_conf.logger),
-                  // then update the agent logger.
-                  agent.logger = agent._conf.logger = logging.createLogger(value)
-                  agent.logger.info(`Remote config success. Updated logger with new logLevel: ${value}`)
-                }
-                agent.logger.info(`Remote config success. Updated ${key}: ${value}`)
-              } else {
-                agent.logger.warn(`Remote config failure. Invalid value for ${key}: ${value}`)
+              const oldValue = agent._conf[key]
```

Comment:
I see here we're dropping the use of the `numberBetweenZeroAndOne` validator from `VALIDATORS`.  I presume this is because this validation has been moved to `normalizeTransactionSampleRate` 👍

Reply:
>  I presume this is because this validation has been moved

Yup. The two separate stages: validation and normalization weren't necessary.